packages/discovery-react-components/src/components/DocumentPreview/components/SimpleDocument/SimpleDocument.tsx

```diff
@@ -34,6 +35,7 @@ export const SimpleDocument: FC<Props> = ({
 }) => {
   const contentRef = useRef<HTMLDivElement>(null);
   const highlightRef = useRef<HTMLDivElement>(null);
+  const { componentSettings } = React.useContext(SearchContext);
```

Comment:
could just import `useContext` above from react instead of doing `React.` -> i think this stuff only affects storybook occasionally but probably doesn't matter here